modin/pandas/test/test_io.py

```diff
@@ -96,6 +97,38 @@ def setup_clipboard(row_size=NROWS):
     df.to_clipboard()
 
 
+def parquet_eval_to_file(modin_obj, pandas_obj, fn, extension, **fn_kwargs):
+    """Helper function to test `to_parquet` method.
+
+    Parameters
+    ----------
+        modin_obj: pd.DataFrame
+            Modin DataFrame or Series to test `to_parquet` method.
+        pandas_obj: pandas.DataFrame
+            Pandas DataFrame or Series to test `to_parquet` method.
+        fn: str
+            Name of the method, that should be tested.
+        extension: str
+            Extension of the test file.
+    """
```

Comment:
```suggestion
    """
    Helper function to test `to_parquet` method.

    Parameters
    ----------
    modin_obj : pd.DataFrame
        Modin DataFrame or Series to test `to_parquet` method.
    pandas_obj: pandas.DataFrame
        Pandas DataFrame or Series to test `to_parquet` method.
    fn : str
        Name of the method, that should be tested.
    extension : str
        Extension of the test file.
    """
```
Since we do not check test files for docstrings CI passes but the docstring doesn't correspond yet to our guide. We should probably fix all docstrings in tests at some point.